api/src/models/survey-create.ts

```diff
@@ -0,0 +1,65 @@
+import { getLogger } from '../utils/logger';
+
+const defaultLog = getLogger('models/survey-create');
+
+/**
+ * Processes all POST /project/{projectId}/survey/create request data.
+ *
+ * @export
+ * @class PostSurveyObject
+ */
+export class PostSurveyObject {
+  survey_name: string;
+  biologist_first_name: string;
+  biologist_last_name: string;
+  foippa_requirements_accepted: boolean;
+  species: string;
+  start_date: string;
+  end_date: string;
+  survey_area_name: string;
+  survey_data_proprietary: boolean;
+  survey_purpose: string;
+  surveyProprietor?: PostSurveyProprietorData;
```

Comment:
kind of a nit but should we stick with snake_case here?

call this `survey_proprietor`?